addOns/help/src/main/javahelp/contents/releases/2_9_0.html

```diff
@@ -62,7 +62,13 @@ <H2>Changes in Bundled Libraries</H2>
 
 
 <H2>Enhancements</H2>
-
+It's now possible to filter requests in Active Scan. Below are the supported criteria's:
```

Comment:
This should be added in the above section (after `Session Management Scripts`?), this section is used for listing the enhancements from the issue tracker.

Reply:
what should be the heading ?
```html <H2>Filtering requests in Active Scan</H2>```

Reply:
I didn't got your point about what to mention in enhancement section ? are you saying the git issue number is mentioned in that section ?

Reply:
Could be just `Active Scan Filter`?

Yes, that's filled before the release with all issues (e.g. https://github.com/zaproxy/zap-core-help/wiki/HelpReleases2_8_0#enhancements) for now it's left empty.

Reply:
ok thanks. done the changes.